Blockzilla/Assets/ContextMenu.js

```diff
@@ -1,166 +1,196 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
 (function() {
 
-"use strict";
+    "use strict";
```

Comment:
@boek can we avoid whitespace changes, unless you feel the spacing was wrong prior?